src/common/alertmanager/alerts/CriticalMissedProposes.ts

```diff
@@ -8,7 +8,9 @@ import { RegistrySourceOperator } from 'validators-registry';
 
 import { Alert, AlertRequestBody, AlertRuleResult } from './BasicAlert';
 
-const VALIDATORS_WITH_MISSED_PROPOSALS_COUNT_THRESHOLD = 1 / 3;
+const validatorsWithMissedProposalsCountThreshold = (quantity: number) => {
+  return Math.min(quantity / 3, 1000);
+};
```

Comment:
shouldn't be changed. In general, there are only 30-40 proposals in 12h frame for huge NO (10k validators, for example), so there is no point in choosing between 1\3 and 1000 here, because 1\3 will always be less than 1000

Reply:
OK, thanks, I agree. I've corrected this.